rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -89,7 +89,7 @@ BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 # Generic requirements
 BuildRequires: gcc bison flex pcre-devel autoconf automake libtool
 Requires: pcre
-Provides: rudder-agent
+Provides: Rudder::Agent
```

Comment:
Shouldn't this "Provides" line also include %{version}? My understanding is that we will hit the same problem again if nto.